runelite-mixins/src/main/java/net/runelite/mixins/RSClientMixin.java

```diff
@@ -1054,4 +1055,18 @@ public static void updateNpcs(boolean var0, PacketBuffer var1)
 		// have been processed is typically more useful.
 		shouldProcessGameTick = true;
 	}
+
+	@Inject
+	@FieldHook("focused")
```

Comment:
Any reason why this isnt a methodhook like before?

Reply:
No reason for it to be methodHook when that field is properly modified in both focusLost and focusGained, this felt like simpler approach.